server/src/main/scala/com/lnvortex/server/coordinator/VortexCoordinator.scala

```diff
@@ -97,15 +98,17 @@ case class VortexCoordinator(bitcoind: BitcoindRpcClient)(implicit
           ScriptType.WITNESS_UNKNOWN | ScriptType.WITNESS_COMMITMENT |
           ScriptType.WITNESS_V0_SCRIPTHASH =>
         throw new IllegalArgumentException("Unknown address type")
-      case ScriptType.PUBKEY             => inputBase + 108 // todo check witness discount
-      case ScriptType.PUBKEYHASH         => inputBase + 108
-      case ScriptType.SCRIPTHASH         => inputBase + 108 + 33
-      case ScriptType.WITNESS_V0_KEYHASH => inputBase + 108
-      case ScriptType.WITNESS_V1_TAPROOT => inputBase + 64
+      case ScriptType.PUBKEY             => 148
```

Comment:
The `scriptSig` for a P2PK input consists only of a signature. 
With outpoint, nSequence and the scriptSig length serialization artifact, that comes out to 32+4+4+1+72 = 113 bytes

```suggestion
      case ScriptType.PUBKEY             => 113
```

Reply:
The other values look good to me.